toolchain/Makefile

```diff
@@ -19,7 +19,7 @@ android-sdk:
 	pushd $(SDK_DIR)/tools/bin && \
 		yes | ./sdkmanager --update && \
 		yes | ./sdkmanager --licenses && \
-		./sdkmanager "platform-tools" "build-tools;30.0.2" "platforms;android-30" "patcher;v4" && \
+		./sdkmanager "platform-tools" "build-tools;30.0.3" "platforms;android-30" "patcher;v4" && \
```

Comment:
Does this require a Docker image update or is it just random?

Reply:
I have no idea how this works in the Docker image, because this is currently handled at runtime, AFAICT?

Reply:
The Docker image will proceed to download the, what is it, 1.5+ GB tools if they're not found but a minor detail like this probably doesn't trigger it. You can tell by what CircleCI does. ;-)

Reply:
But the Docker image comes with them included in a stripped down manner (still at least 600 MB iirc) so it doesn't do that.